impl/src/main/java/org/ehcache/spi/copy/DefaultCopyProvider.java

```diff
@@ -61,15 +62,27 @@ public DefaultCopyProvider(DefaultCopyProviderConfiguration configuration) {
     } else if (conf != null && conf.getClazz().isAssignableFrom(SerializingCopier.class)) {
       copier = new SerializingCopier<T>(serializer);
     } else {
-      copier = createCopier(clazz, conf);
+      copier = createCopier(clazz, conf, serializer);
     }
     LOG.info("Copier for <{}> : {}", clazz.getName(), copier);
     return copier;
   }
 
-  private <T> Copier<T> createCopier(Class<T> clazz, DefaultCopierConfiguration<T> config) {
+  private <T> Copier<T> createCopier(Class<T> clazz, DefaultCopierConfiguration<T> config, Serializer<?> serializer) {
     String alias = (config != null ? null : clazz.getName());
-    Copier<T> copier = (Copier<T>) newInstance(alias, config);
+    Copier<T> copier;
+    if (config == null) {
+      try {
+        copier = (Copier<T>) newInstance(alias, serializer);
+      } catch (RuntimeException e) {
+        if (e.getCause() instanceof InvocationTargetException && e.getCause().getCause() instanceof NullPointerException) {
```

Comment:
This is some ugly binding between the SerializingCopier constructor and this... I think they are already bound... but this one isn't explicit...